src/main/resources/webapp/interface/dynamic/addDynamicObjectTypes.es

```diff
@@ -0,0 +1,142 @@
+
+import {
+	GraphQLBoolean,
+	GraphQLFloat,
+	GraphQLID,
+	GraphQLInt,
+	Json as GraphQLJson,
+	GraphQLString,
+	list,
+	nonNull
+} from '/lib/graphql';
+import {
+	//createConnectionType,
+	//decodeCursor,
+	encodeCursor
+} from '/lib/graphql-connection';
+
+import {
+	GQL_INTERFACE_TYPE_DOCUMENT,
+	GQL_OBJECT_TYPE_AGGREGATIONS_UNION_NAME,
+	GQL_OBJECT_TYPE_INTERFACE_SEARCH,
+	GQL_OBJECT_TYPE_INTERFACE_SEARCH_CONNECTION,
+	GQL_OBJECT_TYPE_INTERFACE_SEARCH_CONNECTION_EDGE,
+	GQL_OBJECT_TYPE_INTERFACE_SEARCH_CONNECTION_PAGE_INFO//,
+	//GQL_OBJECT_TYPE_INTERFACE_SEARCH_HIT,
+	//GQL_OBJECT_TYPE_INTERFACE_SEARCH_HIT_HIGHLIGHT//,
+	//GQL_UNION_TYPE_DOCUMENT_TYPES
+} from '../constants';
+
+
+export function addDynamicObjectTypes({
+	glue//,
+	//interfaceSearchHitsFieldsFromSchema
+}) {
+	/*const interfaceSearchHitsFields = {
+		_collectionId: { type: nonNull(GraphQLID) },
+		_collectionName: { type: nonNull(GraphQLString) },
+		_documentTypeId: { type: nonNull(GraphQLString) },
+		_documentTypeName: { type: nonNull(GraphQLString) },
+		_highlight: { type: glue.getObjectType(GQL_OBJECT_TYPE_INTERFACE_SEARCH_HIT_HIGHLIGHT) },
+		_json: { type: GraphQLJson },
+		_repoId: { type: nonNull(GraphQLString) },
+		_score: { type: GraphQLFloat }
+	};
+	//log.debug(`Object.keys(interfaceSearchHitsFieldsFromSchema):${toStr(Object.keys(interfaceSearchHitsFieldsFromSchema))}`);
+	Object.keys(interfaceSearchHitsFieldsFromSchema).forEach((k) => {
+		interfaceSearchHitsFields[k] = interfaceSearchHitsFieldsFromSchema[k];
+	});
+
+	const objectTypeInterfaceSearchHit = glue.addObjectType({
+		name: GQL_OBJECT_TYPE_INTERFACE_SEARCH_HIT,
+		fields: interfaceSearchHitsFields
+	});*/
```

Comment:
Can you please avoid huge blocks of commented code? It's difficult to read this as it is, and comments don't make it easier.

Reply:
Yeah, I don't clean up much during experimentation. I have to understand the old code, before I delete it (so the new implementation doesn't loose some previously implemented requirement). But yeah cleaning time.